.vscode/tasks.json

```diff
@@ -0,0 +1,61 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build (Jekyll)",
+      "type": "shell",
+      "command": "bundle",
+      "args": [
+        "exec",
+        "jekyll",
+        "build"
+      ]
+    },
+    {
+      "label": "Build (NodeJS)",
+      "type": "shell",
+      "command": "npm",
+      "args": [
+        "run",
+        "build"
+      ]
+    },
+    {
+      "label": "Build and watch (Jekyll)",
+      "type": "shell",
+      "command": "bundle",
+      "args": [
+        "exec",
+        "jekyll",
+        "serve",
+        "--force_polling",
+        "--livereload"
+      ]
```

Comment:
The task label is "Build" here, but the command to run is `jekyll server ...`, so it might be more appropriate to change the label to something like `Start` or `Server`, to distinguish it from `Build`.

Reply:
Changed to `Start watch`.